app/src/UIComponents/Header.css

```diff
@@ -0,0 +1,73 @@
+.header {
```

Comment:
Shouldn't need any of this styling (for now) until we customize our MUI styling to match the UVic colors.

Reply:
Agreed - let's try to avoid making any .css files if at all possible. I'll look into getting us a global Sass file too.